homework4/excercise1.php

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```

Comment:
@Niki1993 
You have done well with the homework :+1: This task is actually from homework 5. Please rename the folders and commit again : )